src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -1,10 +1,14 @@
 .pf-c-modal-box {
   --pf-c-modal-box--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-modal-box--BorderColor: transparent;
-  --pf-c-modal-box--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-modal-box--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-modal-box--PaddingBottom: var(--pf-global--spacer--xl);
-  --pf-c-modal-box--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-modal-box--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-modal-box--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-modal-box--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-modal-box--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-modal-box--m-roomy--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-modal-box--m-roomy--PaddingRight: var(--pf-global--spacer--xl);
```

Comment:
does `pf-m-roomy` mean like spacious? I wonder if Abi would have good insight on the naming here.

Reply:
Yep. @mcarrano did we land on a name or is that a topic for our Friday meeting?